File: freecodecamp/daily-code-challenge/2026-02-24.js
function countBusinessDays(start, end) {
  const startDate = new Date(start);
  const endDate = new Date(end);
  
  let count = 0;
  let current = new Date(startDate);
  
  while (current <= endDate) {
    const day = current.getDay(); // Sunday=0, Saturday=6
    if (day !== 0 && day !== 6) {
      count++;
    }
    current.setDate(current.getDate() + 1);
  }
  
  return count;
}

File: freecodecamp/daily-code-challenge/2026-02-24.py
from datetime import datetime, timedelta

def count_business_days(start, end):
    start_date = datetime.strptime(start, "%Y-%m-%d").date()
    end_date = datetime.strptime(end, "%Y-%m-%d").date()
    
    count = 0
    current = start_date
    
    while current <= end_date:
        if current.weekday() < 5:  # Monday=0, Sunday=6
            count += 1
        current += timedelta(days=1)
    
    return count
